src/main/webapp/app/entities/programming-exercise/programming-exercise-update.component.ts

```diff
@@ -23,6 +21,9 @@ export class ProgrammingExerciseUpdateComponent implements OnInit {
     readonly JAVA = ProgrammingLanguage.JAVA;
     readonly PYTHON = ProgrammingLanguage.PYTHON;
 
+    startedEditing: boolean = false;
```

Comment:
If this does not show a linting error in your IDE, something is not configured correctly.